docs/application/web/api/5.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm

```diff
@@ -254,8 +254,19 @@ <h3>Summary</h3>
 					<div class="returntype">
```

Comment:
It looks like "No Return Value" is written for the case of no return.

However only in this page "No return value" is used in line 209 and line 326.
How about changing "return" to "Return" in this page?